QuantConnect.InteractiveBrokersBrokerage/InteractiveBrokersBrokerage.cs

```diff
@@ -4600,6 +4607,12 @@ private void OnIbAutomaterOutputDataReceived(object sender, OutputDataReceivedEv
                 return;
             }
 
+            if (e.Data.Contains("Waiting for 2FA confirmation", StringComparison.InvariantCultureIgnoreCase))
+            {
+                OnMessage(new BrokerageMessageEvent(BrokerageMessageType.Information, "2FA",
```

Comment:
❌ this wont work in the initialization, since it's called by the brokerage constructor and the instance message callback hasn't been wired